Marsiple/App/Post Details/PostDetailsViewController.swift

```diff
@@ -48,15 +48,22 @@ extension PostDetailsViewController: UITableViewDataSource {
     }
 }
 
+private extension PostDetailsViewController {
+    @objc func commentButtonTapped() {
+        navigationController?.pushViewController(CommentViewController(), animated: true)
+    }
+}
+
 private extension PostDetailsViewController {
     func setupNavigationBar() {
         navigationItem.title = LocalizationKey.PostDetails.navigationBarTitle.localized()
+        let commentButton = UIBarButtonItem.createCommentItem(target: self, action: #selector(commentButtonTapped))
+        navigationItem.rightBarButtonItem = commentButton
```

Comment:
Lets coalesce this into one line - direct assignment, without the dedicated `commentButton` variable.